sympy/functions/elementary/exponential.py

```diff
@@ -436,11 +436,29 @@ def _eval_nseries(self, x, n, logx):
         if arg0 in [-oo, oo]:
             return self
         t = Dummy("t")
-        exp_series = exp(t)._taylor(t, n)
+        nterms = n + 2
+        try:
+            cf = Order(arg.as_leading_term(x), x).getn()
+        except NotImplementedError:
+            cf = 0
+        if cf != 0:
+            try:
+                nterms = (n/cf).ceiling()
+                if nterms < 0:
+                    nterms = n
+            except (AttributeError, TypeError):
```

Comment:
Where do these errors come from?

Reply:
I exactly don't know the source but this expression (`nterms = (n/cf).ceiling()`) was causing these error.So I tried using try except block for it.